assets/src/edit-story/components/panels/document/publish/publishTime.js

```diff
@@ -101,7 +101,7 @@ function PublishTime() {
   return (
     <>
       <Row>
-        <DropDownSelect
+        <DropDown
```

Comment:
Does using `DropDown` instead of `DropDownSelect` (which is a child of `DropDown`) cause any issues here?

cc @miina @BrittanyIRL just in case

Reply:
i believe this is because prometheus wanted to do something else with the exposed menu, and so just the select was used. haven't looked super close.

Reply:
It doesn't seem to cause any issues here, however, we indeed needed only the select button here for opening the calendar.
 @spacedmonkey Why use `DropDown` instead of `DropDownSelect` here?

Reply:
Looks like DropDownSelect is not exported from the design system package.

Reply:
Yep, perhaps it should be exported. It seems to work as-is, too, since by default the dropdown is displaying only the select button anyway, however, there's some other logic that's not needed, ideally we'd keep using the select only for avoiding any potential unexpected behaviour.

Would need some investigation before just switching to using `DropDown`.

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/8087/commits/eecb244e0bc64ee01035f6d538a6e57c6e0cc59a